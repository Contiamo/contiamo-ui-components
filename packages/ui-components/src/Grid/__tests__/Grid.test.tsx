import * as React from "react"
import { render } from "enzyme"

import ThemelessGrid from "../Grid"
import wrapDefaultTheme from "../../../utils/wrap-default-theme"

const Grid = wrapDefaultTheme(ThemelessGrid)

describe("Grid Component", () => {
  it("Should render", () => {
    const renderedComponent = render(<Grid value="SomeValue" />)
    expect(renderedComponent).toMatchSnapshot()
  })
})
