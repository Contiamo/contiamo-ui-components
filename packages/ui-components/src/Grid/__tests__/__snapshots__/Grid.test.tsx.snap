// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid Component Should render 1`] = `
<div
  class="css-1o9b3fq"
/>
`;
